geode-docs/tools_modules/gfsh/command-pages/import.html.md.erb

```diff
@@ -23,79 +23,103 @@ limitations under the License.
 
-| Name                                                  | Description                                                                                     |
-|-------------------------------------------------------|-------------------------------------------------------------------------------------------------|
-| <span class="keyword parmname">\\-\\-zip-file-name</span> | *Required.* Name of the zip file containing the cluster configuration artifacts to be imported. |
+See [export data](export.html#topic_263B70069BFC4A7185F86B3272011734) for an explanation of the difference between serial and parallel data exports.
 
-<span class="tablecap">Table 2. Import Cluster-Configuration Parameters</span>
-
-**Example Commands:**
+If the data was exported from a partitioned region in parallel format, use the `--parallel` option to import the data:
 
 ``` pre
-gfsh>import cluster-configuration --zip-file-name=/home/username/myClusterConfig.zip
+import data --parallel --region=value --member=value --dir=value [--invoke-callbacks=value]
 ```
 
-**Sample Output:**
+In this scenario, partitioned region data is imported simultaneously on all hosting nodes from directories local to those respective nodes.
+
+- Parallel import applies only to partitioned regions exported in parallel mode.
+
+- The `--file` and `--dir` options are mutually exclusive; `--file` can be used only for serial import, `--dir` can be used for both serial and parallel imports.
+
+
+**Import Data Parameters:**
+
+| Name                                           | Description                                                                                                                                                                                                                      | Default Value |
+|------------------------------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|----|
+| <span class="keyword parmname">\\-\\-region</span> | *Required.* Region into which data will be imported. | |
+| <span class="keyword parmname">&#8209;&#8209;member</span> | *Required.* Name/ID of a member that hosts the region. In a serial import, all data is imported from the specified file on the host where the member is running.  In a parallel import, data from a partitioned region partially hosted on this member is imported for each partition from files on the nodes that host those partitions. | |
+| <span class="keyword parmname">\\-\\-file </span>  | File from which the imported data will be read. The file must have an extension of ".gfd" and must be on the file system of the member (or accessible to that member via NFS) that is being targeted for the import. Cannot be specified at the same time as \\-\\-dir, cannot be used with &#8209;&#8209;parallel. | |
```

Comment:
Small formatting comment.  The gfsh import command reference page puts the .gfd suffix into a fixed width font (like code).  I like that formatting better than the double quotes used here.